css/style.css

```diff
@@ -67,6 +67,7 @@ nav a:hover {
 	top: 60%;
 	left: 50%;
 	transform: translate(-50%, -50%);
+	-webkit-transform: translate(-50%, -50%);
```

Comment:
Safari/Mobile safari need the vendor prefix -webkit- this is why the lightbox was incorrect positioned.